src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -65,6 +65,11 @@
 short-summary: Update the throughput of the Cassandra keyspace under an Azure Cosmos DB account.
 """
 
+helps['cosmosdb cassandra keyspace throughput migrate'] = """
```

Comment:
could we add some examples for new commands?

Reply:
I added the examples manually, looks like the style and linter checks are failing after doing so. Is there a way to autogenerate these examples ?

Reply:
@Juliehzl  I have reverted the change where I was adding examples, I will take this up in my next PR. Can you please merge this PR today, so that it makes into the upcoming release cycle. There are customers who are looking forward to this change.